src/components/InputCard/InputCard.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { v4 as uuid } from "uuid";
+import "./inputCard.css";
+
+export default function InputCard({ saveToList }) {
+  const [inputPomodoro, setInputPomodoro] = useState({
+    taskName: "",
+    taskDescription: "",
+    taskTime: "",
+    breakTime: "",
+  });
+
+  const appendTaskToList = (e) => {
+    const task = {
+      id: uuid(),
+      name: inputPomodoro.taskName,
+      description: inputPomodoro.taskDescription,
+      timer: inputPomodoro.taskTime,
+      breakTimer: inputPomodoro.breakTime,
+    };
+
+    saveToList(task);
+    setInputPomodoro({
+      taskName: "",
+      taskDescription: "",
+      taskTime: "",
+      breakTime: "",
```

Comment:
Can store this within a constant known as 
```
defaultTaskTemplate={
 taskName: "",
      taskDescription: "",
      taskTime: "",
      breakTime: "",
}
```
Making it easy for you to refer here!!